ansible/playbook.yaml

```diff
@@ -0,0 +1,68 @@
+- name: Update web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+
+  - name: trello_api_key
+    prompt: What is your Trello API Key? 
+
+  - name: trello_api_token
+    prompt: What is your Trello API Token?
+
+  tasks:
+  - name: Install the latest version of Git
+    become: yes
+    ansible.builtin.yum:
+      name: git
+      state: latest
+
+  - name: Install the latest version of Python
+    become: yes
+    ansible.builtin.yum:
+      name: python311
+      state: latest
+
+  - name: Installing Poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+    args:
+      creates: /home/ec2-user/.local/bin/poetry
+
+  - name: Create Application Folder
+    become: yes
+    ansible.builtin.shell: mkdir /opt/todoapp
+    args:
+      creates: /opt/todoapp
+
+  - name: Change Owner to ec2-user
+    become: yes
+    ansible.builtin.shell: chown ec2-user /opt/todoapp
```

Comment:
These work nicely, but worth noting we could use [the Ansible file module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html) to handle this for us:
```yml
- name: Create Application Folder
  become: yes
  ansible.builtin.file:
    path: /opt/todoapp
    state: directory
    mode: '0755'
    owner: ec2-user
```